client/reader/sidebar/style.scss

```diff
@@ -255,12 +256,13 @@
 				}
 			}
 			.sidebar__menu-icon.sidebar_svg-manage-subscriptions {
+				fill: var(--color-sidebar-gridicon-hover-fill);
```

Comment:
A potential area for improvement here could be to make new semantic variable names that fit the styles they are being applied to.  This is one example where we are using a `*-hover-*` variable for a state that isn't a hover state.  Even if we want the same color, we could define a new variable like `--color-sidebar-gridicon-selected-fill` to use here and set it equal to `--color-sidebar-gridicon-hover-fill` where we define it.  That way in the future if we have a design request to update colors it will just be that much easier to target specifics at a single definition level, separating hover and selected state colors would be a one line change where as here we would need to 'hunt down' all these items 1 by 1 again as you have done here.

Reply:
Good idea, I'll add that change now.